src/lib/wallet/__tests__/GoodWallet.js

```diff
@@ -2,6 +2,7 @@ import goodWallet from '../GoodWallet'
 
 describe('Wallet Initialization', () => {
```

Comment:
What about testing the wallets creation?

Reply:
i've splitted the first 4 initialization tests to be creation tests.